trajectory_publisher/launch/trajectory_setpointraw.launch

```diff
@@ -1,24 +1,4 @@
 <launch>
-  <arg name="mav_name" default="iris"/>
-  <arg name="fcu_url" default="udp://:14540@127.0.0.1:14557"/>
-  <arg name="gcs_url" default="" />
-  <arg name="tgt_system" default="1" />
-  <arg name="tgt_component" default="1" />
-  <arg name="command_input" default="2" />
-  <arg name="gazebo_simulation" default="true" />
-  <arg name="visualization" default="true"/>
-  <arg name="log_output" default="screen" />
-  <arg name="fcu_protocol" default="v2.0" />
-  <arg name="respawn_mavros" default="false" />
```

Comment:
I am not sure why you would want to get rid of the parameter arguments. It will be more visible if someone wants to reconfigure